roles/remove_stale_lun/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+## https://github.com/ansible/ansible/issues/22397
+## Ansible 2.3 generates a WARNING when using {{ }} in defaults variables of role
+## this workarounds it until Ansible resolves the issue:
+- name: Initialize variables
+  set_fact:
+    data_center: "{{ data_center | mandatory }}" 
+    lun_wwid: "{{ lun_wwid | mandatory }}"
+    provided_token: "{{ engine_token | default(lookup('env','OVIRT_TOKEN')) | default('') }}"
```

Comment:
What is the usage of `provided_token`?

Reply:
It's used in log out task. I copy&paste from other roles, so I'm not sure if it's really needed or not.

Reply:
Interesting, you can remove it here and I'll check out the cluster upgrade

Reply:
Removed.